dotCMS/src/main/java/com/dotcms/graphql/business/ContentAPIGraphQLTypesProvider.java

```diff
@@ -134,6 +135,15 @@ private void fillTypesMap() throws DotDataException {
         List<ContentType> allTypes = APILocator.getContentTypeAPI(APILocator.systemUser())
```

Comment:
I am wondering if you implement the hashCode and equals based on the variable you can just use a Set instead of a List to get not dupes.
However may be it is not part of the business requirements